tracer/src/Datadog.Trace/Ci/CIVisibility.cs

```diff
@@ -147,8 +191,8 @@ internal static string GetServiceNameFromRepository(string repository)
 
         internal static IApiRequestFactory GetRequestFactory(ImmutableTracerSettings settings)
         {
-            IApiRequestFactory factory = null;
-            TimeSpan agentlessTimeout = TimeSpan.FromSeconds(15);
+            IApiRequestFactory? factory = null;
+            TimeSpan agentlessTimeout = TimeSpan.FromSeconds(45);
```

Comment:
Can we keep this to 15 and only bump it for ITR requests?

Reply:
done in [36e680b](https://github.com/DataDog/dd-trace-dotnet/pull/3075/commits/36e680b97cdaad987e14d28090fd522eedb7d324)

Reply:
done only for ITR request in https://github.com/DataDog/dd-trace-dotnet/pull/3075/commits/32f45bcfc6bfaab7bb899518e43b7b2000916eaf